src/websocket/index.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import BaseWebSocket, { WsOptions } from './base';
+
+/**
+ * Handles websocket connections and message transmission
+ *
+ * @class
+ * @name WS
```

Comment:
Show we update these annotations?

Reply:
We should, done in 8e3c6f98808c1982f33ba1546e0e985e27e033b9